doc/api/api_changes/2017-07-14-IT_qhull_large_offset.rst

```diff
@@ -0,0 +1,6 @@
+Improved Delaunay triangulations with large offsets
```

Comment:
Could you add a mention of the Matplotlib methods that are affected by this change? Users might not be aware of what a Delaunay triangulation is or what Matplotlib is doing under the hood, but will be able to identify with a specific Matplotlib method that they use.